terraform/modules/data_user_roles/data_cognito_user_roles.py

```diff
@@ -0,0 +1,57 @@
+import sys
+import boto3
+import json
+import math
+from functools import reduce
+
+
+def get_session(role_arn, region):
+    creds = boto3.client('sts').assume_role(
+        RoleArn=role_arn,
+        RoleSessionName="CognitoUserGetIamRoles"
+    )['Credentials']
+
+    return boto3.Session(
+        aws_access_key_id=creds['AccessKeyId'],
+        aws_secret_access_key=creds['SecretAccessKey'],
+        aws_session_token=creds['SessionToken'],
+        region_name='eu-west-2'
```

Comment:
Shouldn't this use the `region` argument?

Reply:
good spot - not sure how I missed that...